test/unit/lualib/symbol.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { Expect, Test, TestCase } from "alsatian";
+import * as util from "../../src/util";
+import { CompilerOptions, LuaLibImportKind } from '../../../src/CompilerOptions';
+
+export class SymbolTests {
+    private compilerOptions: CompilerOptions = {
+        lib: ["esnext"],
+        luaLibImport: LuaLibImportKind.Require,
+    };
+
+    @Test("symbol.toString()")
+    @TestCase()
+    @TestCase(1)
+    @TestCase("name")
+    public symbolToString(description?: string | number): void
+    {
+        const result = util.transpileAndExecute(`
+            return Symbol(${JSON.stringify(description)}).toString();
+        `, this.compilerOptions);
+
+        Expect(result).toBe(`Symbol(${description || ''})`)
+    }
+
+    @Test("symbol.description")
+    @TestCase()
+    @TestCase(1)
+    @TestCase("name")
+    public symbolDescription(description?: string | number): void
+    {
+        const result = util.transpileAndExecute(`
+            return Symbol(${JSON.stringify(description)}).description;
+        `, this.compilerOptions);
+
+        Expect(result).toBe(description);
+    }
+
```

Comment:
Could you add a test that shows symbols are unique, for example `Symbol("foo") === Symbol("foo")` being false?